examples/anonymous-objects/anonymous_objects.description

```diff
@@ -0,0 +1 @@
+// Object types can be defined in-line as well. These types do not have a type name associated with them.
```

Comment:
Can we enhance this some more so that we cover the use case properly here?

Reply:
Addressed with 8722b1c